compiler/quilt/tools/command.py

```diff
@@ -131,7 +131,12 @@ def _save_auth(cfg):
 
 def get_registry_url(team):
     if team is not None:
-        return "https://%s-registry.team.quiltdata.com" % team
+        # Check config
+        cfg = _load_config()
+        if team and 'team_id' in cfg and cfg.get('team_id') == team:
```

Comment:
I meant, `'team_id' in cfg` isn't needed - `cfg.get('team_id') == team` should be sufficient, I think?